templates/src/terraform/aws/modules/emr/main.tf

```diff
@@ -127,16 +127,14 @@ resource "aws_emr_cluster" "sparkling-water-cluster" {
   core_instance_type = "${var.aws_instance_type}"
   core_instance_count = "${var.aws_core_instance_count}"
 
-  tags {
+  tags = {
     name = "SparklingWater"
   }
 
-  bootstrap_action = [
-    {
-      path = "${format("s3://%s/install_sw.sh", aws_s3_bucket.sw_bucket.bucket)}"
-      name = "Custom action"
-    }
-  ]
+  bootstrap_action {
```

Comment:
Have you tried the template with this changed syntax?

Reply:
Yup, I made these change to be able to deploy SW to EMR.

Reply:
Perfect!